src/Install/InstallServiceProvider.php

```diff
@@ -39,7 +42,7 @@ function () {
                         'json',
                         'mbstring',
                         'openssl',
-                        'pdo_mysql',
+                        implode('|', $drivers->getSupportedPhpDrivers()),
```

Comment:
You should probably get the drivers instance from the container instead. Otherwise, there's no point in binding it at all, if it cannot be replaced...